lib/spectacles.rb

```diff
@@ -40,4 +40,13 @@ def trailer(stream)
   end
 end
 
+ActiveRecord::ConnectionAdapters::SchemaStatements.class_eval do
+  alias_method(:_spectacle_drop_table, :drop_table) if method_defined?(:drop_table)
+
+  # Add this so that views get cleaned up properly
+  def drop_table(name, options={})
+    execute "DROP TABLE #{name} CASCADE"
```

Comment:
It looks like [`drop_table`](https://github.com/rails/rails/blob/7785417984f61a9d5e00416c13b89dce2ee02daf/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L375-L386) already supports a `:force => :cascade` option. Perhaps we should simply push that on to the options hash and then let AR do the rest?